File: ArtBahiaAPI/DL - Data Layer/Models/ObraDTO.cs
using Dapper.Contrib.Extensions;

namespace ArtBahiaAPI.Models {
	[Table("Obra")]
	public class ObraDTO {
		[Key]
		public int Id { get; set; }
		public string Titulo { get; set; }
		public int IdUsuario { get; set; }
		public int Tipo { get; set; }
		public int Curtidas { get; set; }
	}
}

File: Art Bahia Android/app/src/main/java/fja/edu/com/artbahia/Models/ObraDTO.java
package fja.edu.com.artbahia.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class ObraDTO {
    public int Id;
    public String Titulo;
    public int IdUsuario;
    public int Tipo;
    public int Curtidas;

    public ObraDTO(){

    }

    public ObraDTO(int id, String titulo, int idUsuario, int tipo, int curtidas){
        this.Id = id;
        this.Titulo = titulo;
        this.IdUsuario = idUsuario;
        this.Tipo = tipo;
        this.Curtidas = curtidas;
    }
}